updating/updating-disconnected-cluster.adoc

```diff
@@ -1,23 +1,134 @@
 [id="updating-disconnected-cluster"]
-= Updating a disconnected cluster to a minor version
+= Updating a disconnected cluster
 include::modules/common-attributes.adoc[]
 :context: updating-disconnected-cluster
 
 toc::[]
 
-You can update, or upgrade, an {product-title} cluster that you installed in a
-restricted network to a minor version by using the web console.
+You can update, or upgrade a disconnected or fully air gapped {product-title} cluster using `oc` commandline.
```

Comment:
nit: "disconnected or fully air gapped" -> "restricted network"?

Also, first references to `oc` in openshift-docs seem to be:

    ... the `oc` command-line interface (CLI).

and similar.  I don't see existing references like:

    ... `oc` commandline.